lib/debug/session.rb

```diff
@@ -2524,7 +2524,7 @@ def daemon(*args)
 
   module TrapInterceptor
     def trap sig, *command, &command_proc
-      case sig&.to_sym
+      case sig&.to_s&.to_sym
```

Comment:
This won't handle integers correctly, because they won't fall into `:INT` and `:SIGINT` below. This would do it:

```ruby
      case sig&.to_s&.to_sym
      when :INT, :SIGINT, :"2"
        if defined?(SESSION) && SESSION.active? && SESSION.intercept_trap_sigint?
          return SESSION.save_int_trap(command.empty? ? command_proc : command.first)
        end
      end
```

Reply:
Sorry I missed your comment. Could you make an issue or PR for it with examples?

Reply:
@ko1 Nevermind, you already fixed this in #976 :)